### ISSUE TITLE
Stop doesn't stop VMs

### ISSUE THREAD

**Issue by jfro** [shapehq/tartelet#91]
### What happened?

0.9.0 & 0.10.0 both exhibiting this, basically I hit stop and no matter if they're mid-boot or booted it doesn't seem to do anything requiring me to just kill the VMs myself by closing their windows.

### What are the steps to reproduce?

Open Tartlet
Start VMs (if they don't auto-start)
Try to stop them via VM menu stop or menu item

### What is the expected behavior?

They should terminate or maybe get killed if graceful termination times out.

**Comment by simonbs** [comment]
This seems like the intended behaviour, where stopping the VMs doesn't actually terminate them immediately. Instead, it prevents Tartelet from starting new VMs once the existing ones are terminated.

This enables Tartelet to gracefully terminate VMs by allowing currently running jobs to complete before. After the VM is stopped, a new one won't be started.

**Comment by jfro** [comment]
ah, guess unexpected and it looks like the UI is trying to stop by saying "stopping" maybe there should be a different way to ahrd stop? or renaming it? i mean it's not awful to just close the 2 windows at least :)

**Comment by simonbs** [comment]
I get why this is unexpected. I think we could get far by chaging the "Stopping..." text. There could also be a "Force Stop" option. Maybe the "Stop" option could change to "Force Stop" when holding down the Option key. It's just a bit dangerous if a job is running.

**Comment by jfro** [comment]
Yeah someway to more clearly label & prevent accidental hard stop sounds good
